packages/components-react/projects/nextjs/components/TableWithCaption.tsx

```diff
@@ -7,14 +7,14 @@ import {
   PTableHeadCell,
   PTableHeadRow,
   PTableRow,
+  Theme,
 } from '@porsche-design-system/components-react/ssr';
 import Image from 'next/image';
 import { headVrt as head, dataVrt as data } from '@porsche-design-system/shared';
 
-// NOTE: this is component is duplicated and adjusted from react
-export const TableWithCaption = (): JSX.Element => {
+const getTableWithCaption = (theme: Theme): JSX.Element => {
```

Comment:
This isn't very react like. Instead the component we had before should be extended by the `theme` prop perhaps with a default value of `light`.